events-amd.js

```diff
@@ -34,9 +34,13 @@ define('event', [], function() {
 	}
 
 	return {
-		addEvent : function(el, eventName, command, named) {
+		addEvent : function(el, eventName, command, named, passive) {
```

Comment:
What about receiving an object with extra configs instead of adding an arg? To keep backwards compatibility, the last argument could be either a string or the object with configs.